src/applications/edu-benefits/feedback-tool/config/form.js

```diff
@@ -179,7 +183,7 @@ const formConfig = {
           title: 'Applicant Information',
           depends: isNotAnonymous,
           uiSchema: {
-            'ui:description': PrefillMessage,
+            'ui:description': data => conditionalPrefillMessage(PREFILL_FLAGS.APPLICANT_INFORMATION, data, PrefillMessage),
```

Comment:
is this `formData`?

Reply:
I'm not sure what the better name for `data` would be. But `formData` is a property _on_ it https://github.com/department-of-veterans-affairs/vets-website/blob/gibft-conditional-prefill/src/applications/edu-benefits/feedback-tool/helpers.js#L306